### DISCUSSION TITLE
Suggestion: Redesign ui with efy ui

### DISCUSSION THREAD

**Discussion by username0136** [FreeTubeApp/FreeTube#2596] [General]
this thing https://github.com/dragos-efy/efy
It will make theming easier and will allow devs to focus on ux
Piped will also use it in future I guess: https://github.com/TeamPiped/Piped/issues/1391

**Comment by ChunkyProgrammer** [comment]
In the Future Plans of that project's ReadMe, it says "Add support for tauri / electron (not a priority, but considered)". FreeTube is an electron app so at least for the time being that does not seem like a good option.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
On another note: We have happily worked with @dragos-efy before in https://github.com/FreeTubeApp/FreeTube/pull/1503. So i have no doubt in my mind that we will see an issue or PR created with mockups when efy is ready for electron.

**Reply by username0136** [reply]
cool :+1:
